index.html

```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
+            <article class="card">
+                <div class="card-work">
+                    <img src="img/Snapshoot Portfolio (2).png" alt="">
+                    <h2 class="title">Tonic</h2>
+                    <div class="client">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tages">
+                        <span class="tage">html</span>
+                        <span class="tage">css</span>
+                        <span class="tage">javaScript</span>
+                    </div>
+                    <button class="normal-button">See Project</button>
+                </div>
+            </article>
+            <article class="card">
+                <div class="card-work">
+                    <img src="img/Snapshoot Portfolio (3).png" alt="">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <div class="client">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tages">
+                        <span class="tage">html</span>
+                        <span class="tage">css</span>
+                        <span class="tage">javaScript</span>
+                    </div>
+                    <button class="normal-button">See Project</button>
+                </div>
+            </article>
+        </div>
+    </section> 
+    <!-- End works -->
+    <!-- Start about me -->
+    <section id="about">
+        <div class="container">
+            <hgroup class="info">
+                <h1>About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </hgroup>
+            <div class="connect">
+                <p>LET’S CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="img/twitter.jpg" alt=""></a></li>
+                    <li><a href="#">in</a></li>
+                    <li><a href="#"><img src="img/medium.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="img/github.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="img/angellist.jpg" alt=""></a></li>
+                </ul>
+            </div>
+            <button class="normal-button">Get my resume</button>
```

Comment:
- [ ] [REQUIRED] Kindly ensure your button follows the template in all states.

| Your Design |
|--------------|
|![image](https://user-images.githubusercontent.com/62938735/191941608-6db83d73-d8c1-417f-b7fb-4796db0d061d.png)|

- [ ] [REQUIRED] Kindly ensure the button. is well aligned to the left